src/docs/product/crons/getting-started.mdx

```diff
@@ -91,10 +91,16 @@ $ curl -X PUT \
 
 : **Required.** The status of your job execution and whether it was completed successfully or unsuccessfully. The values accepted are `in_progress`, `ok` or `error`.
 
+`checkin_id`:
+
+: **Required.** The check-in id to use when updating a check-in to mark the completion or failure of a scheduled job. You can also supply "`latest`" for this parameter in order to retrieve the most recent check-in, more info [here](/api/crons/update-a-checkin/)
+
 `duration`:
 
 : **Optional.** The runtime of the job in milliseconds. If this value is not provided, a duration will be automatically calculated based on the amount of time elapsed from check-in creation to updating a check-in as successful or failed.
 
+More API docs can be found [here](/api/crons/)
```

Comment:
```suggestion
More API docs can be found in [the API documentation](/api/crons/).
```

Reply:
try to avoid links that are just `here` or other contextual words I think, imagine you didn't read the sentence before, does the link still make sense on its own then

Reply:
oooh gotcha gotcha